internal/gitserver/gitdomain/exec.go

```diff
@@ -60,6 +63,44 @@ var (
 	}
 )
 
+var gitObjectRe = regexp.MustCompile(`^[a-fA-F\d]*$`)
```

Comment:
probably can be this?

```suggestion
var gitObjectHashRegex = regexp.MustCompile(`^[a-fA-F\d]*$`)
```